profile/README.md

```diff
@@ -1,5 +1,27 @@
-# SCoverage - Scala Code Coverage Tools
+# scoverage - Scala Code Coverage Tools
 
-Welcome to `SCoverage`, a collection of tools to measure code coverage for [Scala][] code.
+![report][]
 
-[Scala]: https://www.scala-lang.org/
+Welcome to `scoverage`, a collection of tools to measure code coverage for [scala][] code.
+
+`scoverage` features/repositories ...
+
+- [scalac-scoverage-plugin][] - the compiler plugin for Scala 2 and lots of other utilities (e.g. the report writers) that are used by the tools
+- [sbt-scoverage][] - the [sbt][] plugin
+- [gradle-scoverage][] - the [gradle][] plugin
+- [scoverage-maven-plugin][] - the [maven][]
+- [sbt-coveralls][] - the sbt plugin to upload scoverage reports to [coveralls][]
+- [sbt-scoverage-examples][] - example reports, which show how to use scoverage
+
```

Comment:
I know this is all the repos contained in the org, but do you think we should also find a way to link to the Mill support, which is directly in Mill?

Reply:
Yes. Good point. Done.